bcgov/cas-registration
# 392 button tech debt

-392

To Test:

- Login as bc-cas-dev
- start operation 3 report
- Freely navigate back or Save & Continue through the form
- Buttons should all work and disable if the page is read only, they should be at the bottom and change display based on saving state or redirecting state
- *Note: allocation of emissions page is not completed at the time of submitting this PR
